spacelift/resource_space_test.go

```diff
@@ -0,0 +1,54 @@
+package spacelift
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestSpaceResource(t *testing.T) {
```

Comment:
Is it maybe worth adding a test for creating a space, and then creating a child space based on it? In theory there shouldn't be anything special about that, but it seems like a realistic scenario.

Reply:
Sure, will add that.